### ISSUE TITLE
Problem with custom pk (id) field - IntegrityError: NOT NULL constraint failed: *_ptr_id

### ISSUE THREAD

**Issue by lukaszbanasiak** [jazzband/django-polymorphic#103]
I get an error when saving the model with custom `pk` field.

My model:

```
class ModelZ(models.Model):
    id = models.CharField(primary_key=True, max_length=7)
    field0 = models.CharField(max_length=10)
    class Meta:
        abstract = True

class ModelA(PolymorphicModel, ModelZ):
    field1 = models.CharField(max_length=10)

class ModelB(ModelA):
    field2 = models.CharField(max_length=10)
```

Error:

```
ModelB.objects.create(pk='01', field0='09')
[...]
IntegrityError: NOT NULL constraint failed: myapp_modelb.modela_ptr_id
```

Without custom `pk` field in `ModelZ` I can save object.
From admin page I can add an objects, there are no errors.

Django==1.6.5
django-polymorphic==0.5.6

**Comment by ParikhKadam** [comment]
Can anyone help me understand the use of *_ptr_id field in the child model's database table? Why is it used? Can it be null?

I am new to polymorphic django and recently tried it. This column in created in table but is not being populated. The table for child model stays empty. I thought this might point to the primary key column of the parent object, so I manually added and entry to check it out and it fails with errors.

In short, I do not know the use of it and can't find related documentation. Still trying..
